### ISSUE TITLE
Variable declared as Global are accessible from procedure, it shouldn't be the case

### ISSUE THREAD

**Issue by collarbe** [TypeCobolTeam/TypeCobol#938]
This code does not generate errors : 

```COBOL
  01 MyGroup GLOBAL.
         05 zedzedz PIC X(10).


       PROCEDURE DIVISION.

       DECLARE procedure MyProcedure PRIVATE
         INPUT  param1    PIC 9(04)
         OUTPUT outParam1 PIC X(50).
         DATA DIVISION.

         PROCEDURE DIVISION.

           move MyGroup to outParam1.

       END-DECLARE.
```

The variable MyGroup should not be accessible inside the Procedure.

**Comment by smedilol** [comment]
Wait for #995.
Our users will give us feedback about visibility refactoring.

**Comment by fm-117** [comment]
Closed with PR #1468.
